Description: Replace UserManager/TenantManager with None in tests
 UserManager/TenantManager seems private classes in keystoneclient and the
 interface was changed in keystoneclient 0.10.0. django-openstack-auth tests
 actually don't need these managers, so this commit makes test data not use
 these classes.
Origin: upstream, https://review.openstack.org/110050
Author: Akihiro Motoki <motoki@da.jp.nec.com>
Date: Tue, 29 Jul 2014 01:21:40 +0900
Change-Id: I9398c705acab262e89d4801da84ccea188980d99
Bug-Ubuntu: https://bugs.launchpad.net/django-openstack-auth/+bug/1349485
Bug-Debian: https://bugs.debian.org/763253

--- a/openstack_auth/tests/data_v2.py
+++ b/openstack_auth/tests/data_v2.py
@@ -53,8 +53,7 @@
                  'password': 'swordfish',
                  'token': '',
                  'enabled': True}
-    test_data.user = users.User(users.UserManager(None),
-                                user_dict, loaded=True)
+    test_data.user = users.User(None, user_dict, loaded=True)
 
     # Tenants
     tenant_dict_1 = {'id': uuid.uuid4().hex,
@@ -65,12 +64,8 @@
                      'name': 'tenant_two',
                      'description': '',
                      'enabled': False}
-    test_data.tenant_one = tenants.Tenant(tenants.TenantManager(None),
-                                          tenant_dict_1,
-                                          loaded=True)
-    test_data.tenant_two = tenants.Tenant(tenants.TenantManager(None),
-                                          tenant_dict_2,
-                                          loaded=True)
+    test_data.tenant_one = tenants.Tenant(None, tenant_dict_1, loaded=True)
+    test_data.tenant_two = tenants.Tenant(None, tenant_dict_2, loaded=True)
 
     nova_service = {
         'type': 'compute',
